tests/TestCase.php

```diff
@@ -3,30 +3,23 @@
 namespace HotwiredLaravel\TurboLaravel\Tests;
 
 use HotwiredLaravel\TurboLaravel\Facades\TurboStream;
-use HotwiredLaravel\TurboLaravel\TurboServiceProvider;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Orchestra\Testbench\Concerns\WithWorkbench;
 use Orchestra\Testbench\TestCase as Orchestra;
-use Workbench\App\Providers\WorkbenchAppServiceProvider;
 
 class TestCase extends Orchestra
 {
+    use RefreshDatabase;
+    use WithWorkbench;
+
     protected function setUp(): void
     {
         parent::setUp();
 
-        $this->loadMigrationsFrom(dirname(__DIR__).'/workbench/database/migrations/');
-
         TurboStream::fake();
     }
 
-    protected function getPackageProviders($app)
-    {
-        return [
-            TurboServiceProvider::class,
-            WorkbenchAppServiceProvider::class,
-        ];
-    }
```

Comment:
Both `getPackageProviders` and `loadMigrationsFrom` will rely on `testbench.yaml` values when we use `Orchestra\Testbench\Concerns\WithWorkbench`

Reply:
Oh, nice!